alf/trainers/policy_trainer.py

```diff
@@ -246,6 +310,11 @@ def _markdownify(paragraph):
             alf.summary.text('diff', _markdownify(git_utils.get_diff()))
             alf.summary.text('seed', str(self._random_seed))
 
+            # Save a rendered directed graph of the algorithm to the root
+            # directory.
+            _visualize_alf_tree(self._algorithm).render(
```

Comment:
Do you think it's better to put this image in Tensorboard instead?

Reply:
Should be able to do that on top of saving it to the `root_dir`. I could not find a way to directly render graphviz to tensor, so the procedure has to be saving it to the disk, load the image as tensor and summarize it. I'll give it a try tomorrow.